### ISSUE TITLE
Compile-time test uses Android 4 dep incorrectly

### ISSUE THREAD

**Issue by WonderCsabo** [excilys/androidannotations#1430]
Copied from https://github.com/excilys/androidannotations/pull/1427#issuecomment-105154005:

The POM defines 4.1.1.4 version of the Android dependency. However, in our setup, we also add the eclipse-dependencies project to the classpath, which brings android 1.6 dependency, and overrides 4.1.1.4. That is why Eclipse cannot find the newer Activity methods. Maven does, because we do not have the eclipse-dependencies in Maven builds obviously.

Actually the POM also defined the 1.6 version Android dep, but 1b9f70e changed it to 4.1.1.4. I think it was a bad idea, because in our tests, we simulate we do not have Fragment in classpath, then we simulate we have it and add it manually. If we have 4.1.1.4 in the test classpath, we cannot simulate the cases without Fragment, and adding that manually makes no sense. @yDelouis, you are the author of that commit, i think we should revert the version incrementation. That means we have to change some tests, though.

**Comment by dodgex** [comment]
@WonderCsabo any idea if this is still relevant since the eclipse-dependencies are gone?

**Comment by WonderCsabo** [comment]
Yes, it could be relevant, but i am not sure how we could fix it... The problem is that if we remove the optional dependencies from the classpath, the tests will not compile. Well, we could move the test annotated classes to the `resources` folder as well. 😕

**Comment by gauravsak** [comment]
@WonderCsabo @dodgex Is this still a known issue? If not, please close it. Thanks!

**Comment by WonderCsabo** [comment]
It is a know issue, but it does not effect users.
